app/helpers/media_helper.rb

```diff
@@ -105,7 +105,6 @@ def medium_border(medium)
 
   def media_sorts_select(purpose)
     return add_prompt(Medium.select_quizzables) if purpose == "quiz"
-    return Medium.select_question if purpose == "clicker"
```

Comment:
If this is removed, do we still need `Medium.select_question` somewhere?

Reply:
And also what about `app/models/question`?

Reply:
> And also what about `app/models/question`?

What do you mean? Quizzes are made up of questions and remarks. Without questions, there are no quizzes. Clickers were just one very specific way to access questions.

Reply:
ah ok, yeah got confused about the role of question, but not it's clear thanks